odinmain/src/main/kotlin/me/odinmain/features/impl/floor7/DragonPriority.kt

```diff
@@ -28,7 +34,19 @@ object DragonPriority {
         PlayerUtils.alert("§${dragon.colorCode} ${dragon.name}")
     }
 
-    private fun sortPriority(spawningDragon: MutableList<WitherDragonsEnum>): WitherDragonsEnum {
+    /** fun tracerDragonPriority() {
```

Comment:
Having the function here would be cleaner.

Reply:
I agree, however when p5 started it would kick. Moving it to its own class fixed this.